db/hooks.php

```diff
@@ -27,7 +27,7 @@
 $callbacks = [
     [
         'hook' => \core\hook\output\before_http_headers::class,
-        'callback' => [auth_saml2\local\hooks\output\before_http_headers::class, 'callback'],
+        'callback' => 'auth_saml2\local\hooks\output\before_http_headers::callback',
```

Comment:
Gosh, that is true, thanks @paulholden. I knew that it is recent in fact, but did not take into account, oh shame shame.